exe/kubernetes-run

```diff
@@ -5,7 +5,7 @@ require 'kubernetes-deploy/runner_task'
 require 'kubernetes-deploy/options_helper'
 require 'optparse'
 
-template = "task-runner-template"
+template = nil
```

Comment:
The `exe/kubernetes-*` tasks need to remain backwards compatible.

Reply:
Updated the PR.  `exe/kubernetes-run` remains as it is. The `--template` is changed to be required in CLI.